solr/core/src/java/org/apache/solr/handler/IndexFetcher.java

```diff
@@ -261,22 +260,23 @@ public String getMessage() {
     }
   }
 
-  private static HttpClient createHttpClient(
-      SolrCore core,
-      String httpBasicAuthUser,
-      String httpBasicAuthPassword,
-      boolean useCompression) {
+  private Http2SolrClient createHttpClient(
```

Comment:
rename to "createSolrClient" so we don't confuse a lower level httpClient with a solrClient.  Whenever I see a method/field/param name saying "httpClient" but the type is actually "SolrClient"; it irks me.  httpSolrClient is fine if you prefer.